mms/tests/integration_tests/onnx_export_multiple_model_integ_test.py

```diff
@@ -0,0 +1,44 @@
+import json
+import os
+import shutil
+import subprocess
+import tarfile
+from threading import Thread
+import pytest
+import sys
+import time
+import signal
+import test_utils as utils
+try:
+    from urllib2 import urlopen, URLError, HTTPError
+except:
+    from urllib.request import urlopen, URLError, HTTPError
+
+from mms import mxnet_model_server
+
+
+
+### models from onnx-mxnet model zoo
+onnx_model_URLs = {
```

Comment:
urls not URLS

Reply:
some models are commented out. why?

Reply:
this dict of onnx models is repeated in many tests. move to common file, and then filter it when some are incompatible with a particular test

Reply:
Some models were deleted to make load time smaller for multiple models and hence reduce sleep time